pkgs/os-specific/linux/mmc-utils/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation {
 
   meta = with lib; {
     description = "Configure MMC storage devices from userspace";
-    homepage = "http://git.kernel.org/cgit/linux/kernel/git/cjb/mmc-utils.git/";
+    homepage = "http://git.kernel.org/pub/scm/linux/kernel/git/cjb/mmc-utils.git/";
```

Comment:
this repo seems to have moved to https://git.kernel.org/pub/scm/linux/kernel/git/cjb/mmc-utils-old.git, apparently without a replacement/new version anywhere

Reply:
https://git.kernel.org/pub/scm/utils/mmc/mmc-utils.git/ looks like the replacement to me — under a generic name rather than a user account, and updated more recently.  Maybe we should change the package's upstream?

Reply:
that sounds like a good idea :+1: